### ISSUE TITLE
WidgetInspector overlay is not correct when the parent is a Transform

### ISSUE THREAD

**Issue by davidmartos96** [flutter/flutter#59566]
## Steps to Reproduce

Use the select widget option in DevTools when the `WidgetInspector` derived from a `MaterialApp` is inside a `Transform` that scales down for example.

```dart
class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Transform.scale(
      scale: 0.5,
      child: MaterialApp(
        home: MyHomePage(),
      ),
    );
  }
}
```

**Expected results:** 
The widget dimensions overlay should match the `MaterialApp` widget 

**Actual results:** 
The overlay is distortioned so the dimensions do not match.

Is not very common to use a `Transform` widget above a `MaterialApp`, but when using a package such as [device_preview](https://pub.dev/packages/device_preview), where one can test multiple devices at the same time during develop, one can encounter with this issue.

Sample image taken from an issue in the device preview package: [#13](https://github.com/aloisdeniel/flutter_device_preview/issues/13)

![image](https://user-images.githubusercontent.com/22084723/84795109-5d943b80-aff7-11ea-94e9-ff002ea2478e.png)

**Comment by iapicca** [comment]
Hi @davidmartos96 , 
to report bugs or feature request regarding devtools please open an issue in the dedicated [github](https://github.com/flutter/devtools/issues).
Closing, as this isn't an issue with Flutter itself. If you disagree, please write in the comments and I will reopen it.
Thank you

**Comment by davidmartos96** [comment]
@iapicca Hello! I would disagree. The problem is in the Flutter `WidgetInspector` widget, which is inside the flutter repository. 
I've actually went ahead and tried to fix it in this PR #60990 . It's currently pending for review.
